configuration/ldap_config.py

```diff
@@ -71,6 +71,7 @@ def import_group_type(group_type_name):
 
 # For more granular permissions, we can map LDAP groups to Django groups.
 AUTH_LDAP_FIND_GROUP_PERMS = os.environ.get('AUTH_LDAP_FIND_GROUP_PERMS', 'True').lower() == 'true'
+AUTH_LDAP_MIRROR_GROUPS = os.environ.get('AUTH_LDAP_MIRROR_GROUPS', '').lower() == 'true'
```

Comment:
If the default is false, please make this explicit:

```suggestion
AUTH_LDAP_MIRROR_GROUPS = os.environ.get('AUTH_LDAP_MIRROR_GROUPS', 'False').lower() == 'true'
```

Reply:
Set the default to None as per https://django-auth-ldap.readthedocs.io/en/latest/reference.html#auth-ldap-mirror-groups